test/parallel/test-http-res-write-after-end.js

```diff
@@ -26,7 +26,10 @@ const http = require('http');
 
 const server = http.Server(common.mustCall(function(req, res) {
   res.on('error', common.mustCall(function onResError(err) {
```

Comment:
Since `common.expectsError` returns a function wrapped in `common.mustCall` that can be used as callbacks, can you simplify this to:

```js
res.on('error', common.expectsError({
  code: 'ERR_STREAM_WRITE_AFTER_END',
  type: Error
}));
```

?

Reply:
Done!